### PR TITLE
Bugs fixed and updated the driver to version 1.24.4

### PR DESCRIPTION
**Pull request by ghost**
### Changes

* `mongo-c-driver` version `1.24.4`
* Format `*.v` for version `0.4.2`
* Bugs fixed

### CODE REVIEW

File: src/Makefile [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
 	cmake -DENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF .. && \
 	make && \
 	make install
 install:
-	apt install libbson-1.0-0 libmongoc-1.0-0
+	apt install -y libbson-1.0-0 libmongoc-1.0-0
+thirdparty:
+	rm -Rf thirdparty/*
+	cp -R /usr/local/include/libbson-1.0 thirdparty
+	cp -R /usr/local/include/libmongoc-1.0 thirdparty
```
Comment (JalonSolov) [inline src/Makefile]: Missing EOL on this line.

Really need CI on this repo.

**Comment by ghost** [comment]
Includes CI, but requires "Actions Repository secrets" (DATABASE_URL) for tests.

For example, with a cluster in Atlas MongoDB:

```mongodb+srv://<username>:<password>@<cluster>.mongodb.net/?retryWrites=true&w=majority```

> The user must have the role of `atlasAdmin@admin`

**Comment by medvednikov** [comment]
Great job! Thanks!

**Comment by ghost** [comment]
I leave it with MongoDB locally with for the CI

**Comment by JalonSolov** [comment]
Please leave this one open.  It'll be good to have this driver up & running again.

**Comment by ylluminate** [comment]
@moixllik why was this closed again?

**Pull request closed without merging**